website/www/site/layouts/index.html

```diff
@@ -63,73 +61,105 @@ <h2 class="pillars-title">
+          <div>
+            <h4>{{ T "home-graphic-language-title" }}</h4>
+            <p>{{ T "home-graphic-language-body" }}</p>
+          </div>
+        </div>
+      </div>
     </div>
   </div>
 {{ end }}
 
-{{/*
-  TODO:
-    This is the implementation of the design.
-    The event sync functionality will be implemented when we have the event page.
-*/}}
 {{ define "calendar-section" }}
   <div class="calendar">
     <h2 class="calendar-title">
       {{ T "home-calendar-title" }}
     </h2>
     <div class="calendar-content">
-      <a href="#">
-        <div class="calendar-card-big calendar-card-big-left">
-            <div class="calendar-category">
-              <p class="calendar-category-tag">{{ T "home-calendar-category-tag-first" }}</p>
-              <p class="calendar-category-date">{{ T "home-calendar-category-date-first" }}</p>
-            </div>
-            <h4 class="calendar-card-big-title">
-              {{ T "home-calendar-card-title-first" }}
-            </h4>
-            <p class="calendar-card-author">{{ T "home-calendar-card-author-first" }}</p>
+      <div class="calendar-card-big calendar-card-big-left">
+        <div class="desktop">
+          <a class="twitter-timeline" data-width="381" data-height="460"  href="https://twitter.com/ApacheBeam?ref_src=twsrc%5Etfw">Tweets by ApacheBeam</a>
         </div>
-      </a>
+        <div class="mobile">
+          <a class="twitter-timeline" data-width="381" data-height="348"  href="https://twitter.com/ApacheBeam?ref_src=twsrc%5Etfw">Tweets by ApacheBeam</a>
+        </div>
+      </div>
       <div class="calendar-card-box">
-        <a href="#">
-          <div class="calendar-card-small">
-            <div class="calendar-category">
-              <p class="calendar-category-tag">{{ T "home-calendar-category-tag-second" }}</p>
-              <p class="calendar-category-date">{{ T "home-calendar-category-date-second" }}</p>
+        {{ range first 2 (after 1 (where .Site.Pages.ByPublishDate.Reverse "Section" "blog")) }}
+            <a class="post-card" href="{{ .RelPermalink }}" data-categories="{{ with .Params.categories }}{{ if eq ( printf "%T" . ) "string" }}{{ . | replaceRE "[.]" "_" | urlize }}{{ else if eq ( printf "%T" . ) "[]string" }}{{ range . }}{{ . | replaceRE "[.]" "_" | urlize }} {{end}}{{end}}{{else}}no-author{{end}}"
+            >
+              <div class="post-info post-category">
+                  <p>
+                    {{ if .Params.categories }}
+                      {{ delimit .Params.categories ", " " & " }}
+                    {{ end }}
+                  </p>
+                  <p>{{ .Date.Format "2006/01/02" }}</p>
+              </div>
+              <div>
+              <p class="post-title">{{ .Title }}</p>
+              <p class="post-info">
+                {{ if .Params.authors }}
+                      {{ delimit .Params.authors ", " " & " }}
+                {{ end }}
```

Comment:
This renders the author ASF id (bhulette). Is it possible to make it show their full name (Brian Hulette)?

![image](https://user-images.githubusercontent.com/675055/106537446-32d7f400-64af-11eb-8fa0-1c28189b2ad8.png)

Reply:
We should also change this in other places where we have these cards. I see them:
- When browsing the blog
- At the bottom of blog posts in the "Latest from the blog" section

Reply:
Yeah no problem, I changed it

Reply:
Thank you, this is much better!

I noticed another issue around this though, in the "Latest from the blog" section we're attributing the blog posts to the author of _this_ post, rather than the authors of the linked post:
![image](https://user-images.githubusercontent.com/675055/106633834-05815980-6534-11eb-9e42-1443a80b7542.png)